1725717681119	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1725720128753	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1725720128777	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1725720128791	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1725720128804	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1725720128865	Sync.SyncScheduler	DEBUG	Woke from sleep.
1725720128866	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1725720128866	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1725720128867	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1725720130869	Sync.Service	DEBUG	User-Agent: Firefox/130.0 (Linux x86_64) FxSync/1.132.0.20240902145003.desktop
1725720130869	Sync.Service	INFO	Starting sync at 2024-09-07 20:12:10 in browser session wV10o-p8OhBY
1725720130869	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1725720130891	Sync.Status	INFO	Resetting Status.
1725720130891	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1725720130893	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1725720130893	FirefoxAccounts	DEBUG	getOAuthToken enter
1725720130894	FirefoxAccounts	TRACE	getCachedToken returning cached token
1725720130894	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1725720130895	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1725720130896	Sync.SyncAuthManager	INFO	Getting sync key
1725720130897	FirefoxAccounts	DEBUG	getOAuthToken enter
1725720130897	FirefoxAccounts	TRACE	getCachedToken returning cached token
1725720130897	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1725720130897	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1725720130897	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1725720130897	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1725720130897	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1725720130901	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1725720130901	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1725720130902	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1725720130902	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1725720130903	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1725720130903	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1725720130903	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1725720130904	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1725720130905	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1725720130905	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1725720130908	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/261848012/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1725720130908	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1725720130908	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1725720130908	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1725720130910	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1725720130911	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1725720130942	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
